backend/db/utils.py

```diff
@@ -54,24 +54,37 @@ def refresh_termhub_core_cset_derived_tables(con: Connection, schema: str):
         'cset_members_items_plus',
         # 'csets_to_ignore',            not using for now
     ]
+
     # Create new tables and backup old ones
+    print('Derived tables')
     t0 = datetime.now()
     for module in ddl_modules:
-        print(f'Running SQL to recreate derived table: {module}...')
+        print(f' - creating new table/view: {module}...')
         statements: List[str] = get_ddl_statements(schema, [module], temp_table_suffix, 'flat')
-        for statement in statements:
+        main_statements = [s for s in statements if not s.startswith('CREATE INDEX')]
+        for statement in main_statements:
             run_sql(con, statement)
         # todo: warn if counts in _new table not >= _old table (if it exists)?
         run_sql(con, f'ALTER TABLE IF EXISTS {schema}.{module} RENAME TO {module}_old;')
         run_sql(con, f'ALTER TABLE {schema}.{module}{temp_table_suffix} RENAME TO {module};')
-    # - Delete old tables. Because of view dependencies, order & commands are different
-    t1 = datetime.now()
-    print(f'Derived tables all created in {(t1 - t0).seconds} seconds. Removing older, temporarily backed up copies...')
+
+    # Delete old tables/views. Because of view dependencies, order & commands are different
+    print(f' - Removing older, temporarily backed up tables/views...')
     for view in views:
         ddl_modules.remove(view)
         run_sql(con, f'DROP VIEW IF EXISTS {schema}.{view}_old;')
     for module in ddl_modules:
         run_sql(con, f'DROP TABLE IF EXISTS {schema}.{module}_old;')
+    t1 = datetime.now()
+
+    # Indexes
+    for module in ddl_modules:
+        print(f' - creating indexes for table/view: {module}...')
+        statements: List[str] = get_ddl_statements(schema, [module], temp_table_suffix, 'flat')
+        index_statements = [s for s in statements if s.startswith('CREATE INDEX')]
```

Comment:
Index DDL: runs last, after new tables have been made and old temporarily backed up tables (and their indexes) are deleted.

Reply:
Changing this now so that we do a hash instead.